### ISSUE TITLE
bug: conflict with @capacitor-community/app-icon plugin

### ISSUE THREAD

**Issue by maxinminax** [Cap-go/capacitor-updater#227]
### Capacitor Version
```
@capacitor/cli: 4.8.0
@capacitor/android: 4.8.0
@capacitor/core: 4.8.0
@capacitor/ios: 4.8.0
```

### Plugin Version
```
@capgo/cli: 3.10.20
@capgo/capacitor-updater: ^4.61.10
```

### context(s)
```md
ManualModel: false
AutoMode: false
CapgoCloud: false
OnPremise: false
```

### Platform(s)
android

### Current Behavior
App will crashed when change activity by @capacitor-community/app-icon plugin
```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.ComponentName.getClassName()' on a null object reference
	at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin.isMainActivity(CapacitorUpdaterPlugin.java:1269)
	at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin.onActivityStopped(CapacitorUpdaterPlugin.java:1289)
```

**Comment by homealerts** [comment]
I'm receiving this same Android error (via pre-launch report SDK 32)

`java.lang.RuntimeException: Unable to stop activity {com.homealertsapp.nest/com.homealertsapp.nest.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.ComponentName.getClassName()' on a null object reference`

**Comment by riderx** [comment]
I tried to catch better the issue in `5.0.12` can you try it ?

**Comment by riderx** [comment]
Oh yes the release was blocked by lint issue, i have send a fix in 5.0.13

**Comment by homealerts** [comment]
5.0.13 resolved the Android pre-launch report errors mentioned above - thanks!
